logfire/_internal/config.py

```diff
@@ -801,16 +801,20 @@ def add_span_processor(span_processor: SpanProcessor) -> None:
             if isinstance(self.metrics, MetricsOptions):
                 metric_readers = list(self.metrics.additional_readers)
 
+            creds_from_file = LogfireCredentials.load_creds_file(self.data_dir)
+            if creds_from_file:
+                self.token = creds_from_file.token
```

Comment:
No, a directly configured token should take priority

Reply:
not just directly, a token configured in any way (env var, pyproject.toml) should take priority over the creds file

Reply:
Makes sense, fixed.